docs/config/README.md

```diff
@@ -48,7 +48,14 @@ minio server ~/Photos
 #### Region
 |Field|Type|Description|
 |:---|:---|:---|
-|``region``| _string_ | `region` describes the physical location of the server. By default it is set to `us-east-1`, which is same as AWS3's default region. If you are unsure leave it to default.|
```

Comment:
s/AWS3/AWS S3/

s/If you are unsure leave it to default./If you are unsure leave it unset./